src/main/java/net/minestom/server/utils/Direction.java

```diff
@@ -15,11 +16,13 @@ public enum Direction {
     private final int normalX;
     private final int normalY;
     private final int normalZ;
+    private final Vec normalVec;
```

Comment:
I think we should remove normalVec

Reply:
Why? it seems ok to me

Reply:
It just seems weird to me to store the same data twice

Reply:
There are 6 of these ever instantiated. It's negligible and avoids a lot of allocation